ui/push-health/CommitHistory.jsx

```diff
@@ -55,93 +61,121 @@ class CommitHistory extends React.PureComponent {
+    const headerText = revisions[0].comments.split('\n')[0];
+    const authorMatch = author.match(/<(.*?)>+/);
+    const authorEmail = authorMatch ? authorMatch[1] : author;
+    const expandIcon = isExpanded ? faCaretDown : faCaretRight;
+    const expandTitle = isExpanded ? 'Click to collapse' : 'Click to expand';
+    const expandText = isExpanded ? 'Hide all commits' : 'Show all commits';
 
     return (
       <React.Fragment>
-        <div className="push-header border-bottom" data-testid="push-header">
+        <div className="push-header" data-testid="push-header">
           <div className="push-bar">
-            <a
-              href={revisionPushFilterUrl}
-              title="View this push in Treeherder"
-            >
+            <div className="h3 text-capitalize">{headerText}</div>
+            <div className="text-secondary">
               {toDateStr(pushTimestamp)}
-              <FontAwesomeIcon
-                icon={faExternalLinkAlt}
-                className="ml-1 icon-superscript"
-              />
-            </a>
-            <span className="mx-1">-</span>
-            <PushAuthor author={author} url={authorPushFilterUrl} />
+              <span className="mx-1">-</span>
+              <span>{authorEmail}</span>
+            </div>
           </div>
-        </div>
-        {revisions.length <= 5 || showAllRevisions ? (
-          <RevisionList
-            revision={revision}
-            revisions={revisions.slice(0, 20)}
-            revisionCount={revisionCount}
-            repo={currentRepo}
-          />
-        ) : (
-          <span>
-            <RevisionList
-              revision={revision}
-              revisions={revisions.slice(0, 5)}
-              revisionCount={revisionCount}
-              repo={currentRepo}
-            />
-            <Button
-              outline
-              color="darker-secondary"
-              onClick={() =>
-                this.setState({ showAllRevisions: !showAllRevisions })
-              }
-            >
-              Show more...
-            </Button>
-          </span>
-        )}
-        <div className="mt-4">
-          Parent Push:
-          <span className="ml-2">
-            {!exactMatch && (
-              <div>
-                <Alert color="warning" className="m-3 font-italics">
-                  Warning: Could not find an exact match parent Push in
-                  Treeherder.
-                </Alert>
-                {id && <div>Closest match: </div>}
-              </div>
-            )}
-            <span
-              className="mb-2"
-              onMouseEnter={() => this.showClipboard(true)}
-              onMouseLeave={() => this.showClipboard(false)}
-            >
-              <Clipboard
-                description="full hash"
-                text={parentSha}
-                visible={clipboardVisible}
-              />
+          <div className="text-secondary">
+            Push{' '}
```

Comment:
Instead of having a space with `{' '}`, try adding the class of `ml-1` to the `<span...` on line 84.

`ml-1` is Bootstrap utilities for "margin-left of 1"  I think it might be `1rem`.